### DISCUSSION TITLE
A clarification regarding Proper Motion info display in Stellarium.

### DISCUSSION THREAD

**Discussion by BikramAjay** [Stellarium/stellarium#1824] [General]
Is it safe to assume the PM component pointed by the red arrow is mu_alpha and the one pointed by the yellow arrow is mu_delta? Or is it other way around? Please see attached screenshot from the Stellarium screen. 

![Screenshot 2021-07-30 122855](https://user-images.githubusercontent.com/86772525/127683577-54b99298-7202-4183-8360-54b8a00fe320.jpg)

**Comment by gzotti** [comment]
Is there any reason to doubt that the canonical order of (alpha, delta) does not apply here?

**Reply by BikramAjay** [reply]
I am new so just confirming.

**Reply by gzotti** [reply]
I also may be wrong. The star catalog has been developed before my involvement. See the User Guide, Appendix C.2 and the code, Star.hpp. @alex-w It seems we are printing proper motions relative to the catalog zone axes, which may not necessarily be the dRA/dDEC values!

**Reply by BikramAjay** [reply]
Thanks for pointing this out.

**Reply by Atque** [reply]
@gzotti Does that only affect the output, or does it affect the graphical displayal of proper motion? Otherwise, we might be able to solve the issues I mentioned in #1787.

**Reply by gzotti** [reply]
Guesswork from reading the public source code: 
The author of the star catalog did his best to create something great and efficient in or around 2006, accepting a few minor issues like binaries not being attached to each other, no 3D motion and magnitude/distance/RV change.
On request of printed values for proper motion, one of the maintainers 10 years later might not have gone into the absolute depths of the code and assumed the delta values stored in the catalog are the requested PM values and added these data lines.
I currently must assume from a series of detailed questions about wrong PM values that the Dx0/Dx1 values are not exactly the deltaRA/deltaDE values, but that the given positions (getJ20000pos() as developed by the original catalog contributor) are the best available approximation within the star catalog geometry. So, we maintainers may have to re-work the star field axes vector geometry to reconstruct dRA/dDE values.

Would please anybody else also get involved and read the code, analyze, and improve it? And then create a new Gaia/Hipparcos catalog without a single star brighter than mag 16 missing, with binaries not flying apart, with 3D proper motion, with no star hiding when it leaves its field? And of course, be efficient and fast. Maybe use 2GB of texture memory and do everything on the GPU? Win11 will force us to throw away all but the latest PCs anyhow, all my current devices included...
